comictaggerlib/taggerwindow.py

```diff
@@ -2065,6 +2097,7 @@ def closeEvent(self, event: QtGui.QCloseEvent) -> None:
                 self.config[0].internal__sort_column,
                 self.config[0].internal__sort_direction,
             ) = self.fileSelectionList.get_sorting()
+            self.config[0].internal__load_data_style = self.cbLoadDataStyle.currentData()
```

Comment:
Why is this added here? why isn't this set in the equivalent place that the save data style is set

Reply:
Reverted.

This was because of the whole "reversed" thing.